DirectProgramming/C++SYCL_FPGA/Tutorials/Features/hls_flow_interfaces/component_interfaces_comparison/sample.json

```diff
@@ -1,7 +1,7 @@
 {
     "guid": "7d8482f5-39f1-4cf1-aa2e-a1f72cfc47cb",
     "name": "Component Interfaces Comparison",
-    "categories": ["Toolkit/oneAPI Direct Programming/C++SYCL FPGA/Getting Started Tutorials"],
```

Comment:
@pmpeter1  is it ok to have multiple categories for a code sample? I would like the sample to show up in two places in the sample browser.

Reply:
@jimmytwei

Reply:
@whitepau Take a look at:
https://github.com/oneapi-src/oneAPI-samples/blob/release/2024.2/DirectProgramming/C%2B%2BSYCL/DenseLinearAlgebra/vector-add/sample.json